src/sensor.cpp

```diff
@@ -2,7 +2,7 @@
 // Copyright(c) 2015 Intel Corporation. All Rights Reserved.
 
 #include "sensor.h"
-
+#include "api.h"
```

Comment:
Please do not mix `api` with internal source code.
This macro is only used on API, and if you search the code you can verify it. 
Please use other methods to do so.. `if (!...)`

Reply:
OK, Done.
Can I leave api usage in the "stream.h" file? I see we have usages of this macro there.

Reply:
1. I still see the include here
2. Where do you see a usage of this macro on `stream.h`?

Reply:
Sorry, not in the `stream.h`, in the `rs.cpp`.